### DISCUSSION TITLE
fediverse/

### DISCUSSION THREAD

**Discussion by ghost** [eallion/eallion.com#96] [Giscus]
# fediverse/

@e5n.cc@eallion 前言：联邦宇宙没有门槛 不要试图去理解它，要去感受它。——《信条》 联邦宇宙 (Fediverse = federation + universe) 也许会给新用户带来高深莫测的感觉，但就像《信条》的这句名

https://www.eallion.com/fediverse/

**Comment by eallion** [comment]
评论暂时迁移至 GitHub Discussions，慢慢会关闭评论。

**Comment by skyue** [comment]
尝试过多次Mastodon，都放弃了。
很喜欢其理念，但总觉得不够简洁优雅，有点复杂。

**Reply by eallion** [reply]
我也是非常喜欢其理念，转了一圈现在又回到了 Mastodon 的怀抱。
为了兼容这兼容那，为了这协议那协议，为了各种 License，这是大型开源项目的通病。

**Comment by ghost** [comment]
感觉 GoToSocial 是一个很好的最小可用版本，兼容了大部分平时会用到的 API。在我的眼中，只要内存占用高于 512M，都是重得无法接受的。

**Reply by eallion** [reply]
就目前来看，确实是最小可用版本，就 20M 上下浮动。主要是不自带发嘟应用。

**Reply by ghost** [reply]
这个应该不算是大的缺点，现在的独立前端蛮多的，也有很多客户端可以用。
偏个题，楼主的硅谷 Lighthouse 是在优惠期间买的吗？我没有看到这个机型。

**Reply by eallion** [reply]
好像是活动的时候买的。
